bin/beaker

```diff
@@ -3,7 +3,11 @@
 require 'rubygems' unless defined?(Gem)
 require 'beaker'
 
-Beaker::CLI.new.execute!
+if (Beaker::Subcommands.instance_methods(false) << :help).include? ARGV[0].to_sym
```

Comment:
Can we give this an intention-revealing name, wrap it in a method and use that instead?